packages/dotcom-build-code-splitting/src/index.ts

```diff
@@ -60,7 +60,15 @@ export class PageKitCodeSplittingPlugin {
       usedInUnknownWay: true
     })
 
-    // split packages used by all pages (i.e. used y Page Kit) into a shared bundle
+    // split all privacy- packages into one bundle file
+    const addPrivacyCodeSplitting = createBundleWithRegExp({
+      compiler,
+      name: 'privacy-components',
+      pattern: /[\\\/]privacy-/,
```

Comment:
I think this pattern is too generic and it would be sensible to prepend it with `@financial-times` to ensure it only matches the intended modules.

Reply:
i'll update the pattern